js/script.js

```diff
@@ -0,0 +1,50 @@
+var searchResults = [];
+var slideshow; 
```

Comment:
Great job leaving variable declarations outside of `$(document).ready`. That's good practice, since the variables can be loaded while the page is still loading.